bindata/network/ovn-kubernetes/ovnkube-master.yaml

```diff
@@ -95,9 +95,9 @@ spec:
           preStop:
             exec:
               command:
-                - OVN_MANAGE_OVSDB=no
-                - /usr/share/ovn/scripts/ovn-ctl
-                - stop_northd
+                - /bin/bash
+                - -c
+                - OVN_MANAGE_OVSDB=no /usr/share/ovn/scripts/ovn-ctl stop_northd
```

Comment:
If this is all we need, then it would be better to avoid the attack surface that is bash by using coreutils `/usr/bin/env` which was designed for this purpose.